### PR TITLE
Admin dashboard

### PR DESCRIPTION
**Pull request by kozlovzxc-at-lido**
## Description

UI for admins to revoke unclaimed tokens

## Demo

Loading data
image

Active & revoked programs
image

## Review notes

What is the best way to test it locally? Just add myself to admins list?

## Testing notes

Need to ask dgusakov to add to vesting managers on testnet

### CODE REVIEW

File: features/admin/vestingRow.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      <Td>{vesting.escrow}</Td>
+      <Td>{vesting.recipient}</Td>
+      <Td>
+        {isRevokedSWR.isLoading ? (
+          <Loader />
+        ) : isRevokedSWR.data ? (
+          'Revoked'
+        ) : (
+          'Active'
+        )}
```
Comment (Jabher) [inline features/admin/vestingRow.tsx:25]: better to move nested ternaries externally, they are hard to read when inlined

File: features/vesting/hooks.ts [hunk 2]
```diff
@@ -173,3 +243,45 @@ export const useVestingSnapshotDelegate = () => {
...
+
+export const useRevokeUnvested = (escrow: string | undefined) => {
+  const { chainId } = useWeb3();
+  const { contractWeb3 } = useVestingEscrowContract(escrow);
+
+  return useCallback(async () => {
+    if (chainId == null || contractWeb3 == null) {
+      return;
+    }
+    await transaction('Revoke unvested tokens', chainId, () =>
```
Comment (Jabher) [inline features/vesting/hooks.ts:255]: you sure you do not need return value here?

File: features/vesting/hooks.ts [hunk 3]
```diff
@@ -173,3 +243,45 @@ export const useVestingSnapshotDelegate = () => {
...
+  const getEvents = useCallback(
+    async (chainId?: CHAINS) => {
+      if (chainId == null) return false;
+
+      const events = await contractRpc.queryFilter(
+        contractRpc.filters.UnvestedTokensRevoked(),
+        EVENTS_STARTING_BLOCK[chainId],
+      );
+
+      return events.length > 0;
```
Comment (Jabher) [inline features/vesting/hooks.ts:274]: this FN should be probably named `getHasSomeEvents`

**Review by Jabher — APPROVED** [review]

File: features/admin/vestingRow.tsx (hunk 1 above)
Comment (kozlovzxc-at-lido) [reply features/admin/vestingRow.tsx:25]: Moved to separate components

File: features/vesting/hooks.ts (hunk 2 above)
Comment (kozlovzxc-at-lido) [reply features/vesting/hooks.ts:255]: It's not used anywhere 🤔

File: features/vesting/hooks.ts (hunk 3 above)
Comment (kozlovzxc-at-lido) [reply features/vesting/hooks.ts:274]: Makes sense

**Review by Jabher — APPROVED** [review]

**Pull request merged**